ember-power-calendar/package.json

```diff
@@ -120,7 +120,6 @@
     "eslint-plugin-import": "^2.30.0",
     "eslint-plugin-n": "^17.10.2",
     "eslint-plugin-prettier": "^5.2.1",
-    "node-sass": "^9.0.0",
```

Comment:
one difference to the other EBD PR - power calendar already has `sass` devdep below, so I could drop node-sass here without issue